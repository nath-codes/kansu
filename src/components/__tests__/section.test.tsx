import React from 'react'

import Section from '../section'
import { renderWithTheme } from './helpers'

describe('Section', () => {
  it('renders correctly', () => {
    const tree = renderWithTheme(
      <Section>
        <h1>Heading</h1>
        <div>
          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Soluta vero
          amet quae laudantium sapiente illum expedita, culpa, nostrum, esse
          tenetur dicta ducimus? At suscipit nam amet perspiciatis veritatis
          repellendus accusantium.
        </div>
      </Section>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
