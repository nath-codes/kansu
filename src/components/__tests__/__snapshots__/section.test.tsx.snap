// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin-bottom: 3rem;
  position: relative;
}

.c0:last-child {
  margin-bottom: 3.75rem;
}

@media only screen and (min-width:768px) {
  .c0 {
    margin-bottom: 4.125rem;
  }

  .c0:last-child {
    margin-bottom: 0;
    padding-bottom: 6.75rem;
  }
}

<section
  className="c0"
>
  <h1>
    Heading
  </h1>
  <div>
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Soluta vero amet quae laudantium sapiente illum expedita, culpa, nostrum, esse tenetur dicta ducimus? At suscipit nam amet perspiciatis veritatis repellendus accusantium.
  </div>
</section>
`;
